kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java

```diff
@@ -63,6 +63,9 @@ public class PatternViewModel extends FormViewModel {
 
     public static String MEANING_DATE_STR = "meaningDateStr";
 
+    // Used to load the values in the PatternField controller from PatternDetailsController.
+    public static String SELECTED_PATTERN_FIELD = "selectedPatternField";
```

Comment:
I thought we discussed this in detail. Selected items for editing or removing does not need to be maintained. The observable list of Fields collection are indexed.

Reply:
@carldea , I think what we discussed was, said that we would not be creating an instance in the controller class and we are okay in passing the actual value around via ViewModel. 
     Hence did the variable creation. This is required when we edit a field and change its field position, for example we created 3 fields, and then we want to edit field 3 and change its field order to 1. In this scenario, we will have to retain either the selected field order or selected pattern field, I chose the later. And then when we come back to the details screen we can use it to remove and then add it to the desired position so that the order is changed. that is Field 3 becomes1, Field 1 becomes 2 and Field 2 becomes 3.

Reply:
After talking with @dholubek he said we can discuss the use of the Axiomic API with you on this implementation.